libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.component.html

```diff
@@ -30,9 +30,10 @@
       class="sky-vertical-tabset-group-container"
       [@tabGroupEnter]="tabService.animationTabsVisibleState"
       [ngClass]="{ 'sky-vertical-tabset-hidden': !tabService.tabsVisible() }"
+      [tabIndex]="tablistHasFocus ? -1 : 0"
```

Comment:
This failed the accessibility test b/c scrollable elements need to be focusable.